### ISSUE TITLE
Missing accessibility labels

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud/calendar#7928]
### Steps to reproduce

Issues:
- The timepickers from start date/end date/start time/ end time have no labels associated with them. At minimum they need an aria-label
- Certain list elements with an input at the end do not have labels for that input, like in attachments:

<img width="679" height="57" alt="Image" src="https://github.com/user-attachments/assets/8aec7e12-b98c-4192-95bc-9c5c48ae8ace" />

- The inputs to the right of editfull have hoverable icons as describers, but they also need proper labels

<img width="338" height="213" alt="Image" src="https://github.com/user-attachments/assets/1435aacb-4cb6-4651-bc36-f0680a3b7c75" />

### Expected behavior

Have proper labels

### Actual behavior

Not accessible or not existing labels

### Web server

None

### Database engine version

None

### PHP engine version

None

### Updated from an older installed version or fresh install

None

### List of activated apps

```Text

```

### Nextcloud configuration

```Text

```

### Web server error log

```Text

```

### Log file

```Text

```

### Browser log

```Text

```
